it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -172,13 +172,13 @@ noIdentities=You don't have any email identities yet.  Create one with the Accou
 12551=Pubblicazione del messaggio in corso…
 
 ## @name NS_ERROR_MSG_MULTILINGUAL_SEND
-12553=The message you composed contains characters not found in the selected Character Encoding. While you can choose a different Character Encoding, it is usually safe to use Unicode (UTF-8) for mail. If you just send in the current Character Encoding, be aware that those not covered by the current Character Encoding will be turned to question marks and illegible by the recipient.
+12553=Il messaggio che hai scritto contiene caratteri non trovati nella Codifica caratteri selezionata. Sebbene tu possa scegliere una Codifica caratteri differente, di solito utilizzare Unicode (UTF-8) è il metodo più sicuro per le e-mail. Se invii il messaggio nella Codifica caratteri corrente, ti avvisiamo che i caratteri non coperti dalla Codifica caratteri corrente si trasformeranno in punti interrogativi e in caratteri illeggibili per il destinatario.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12553=Il messaggio che hai scritto contiene caratteri non trovati nella Codifica caratteri selezionata. Sebbene tu possa scegliere una Codifica caratteri differente, normalmente l'utilizzo di Unicode (UTF-8) è il metodo più sicuro per l'invio delle e-mail. Se invii il messaggio usando la Codifica caratteri corrente, tieni presente che i caratteri non inclusi nella Codifica caratteri corrente si trasformeranno in punti interrogativi e in caratteri illeggibili per il destinatario.